v3/lints/cabf_cs_br/lint_cs_eku_required.go

```diff
@@ -0,0 +1,83 @@
+package cabf_cs_br
+
+import (
+	"fmt"
+
+	"github.com/zmap/zcrypto/x509"
+
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+/* 7.1.2.3 Code signing and Timestamp Certificate
+f. extKeyUsage
+If the Certificate is a Code Signing Certificate, then id-kp-codeSigning MUST be present
+and the following EKUs MAY be present:
+	• Lifetime Signing OID (1.3.6.1.4.1.311.10.3.13)
+	• id-kp-emailProtection
+	• Document Signing (1.3.6.1.4.1.311.3.10.3.12)
+
+If the Certificate is a Timestamp Certificate, then id-kp-timeStamping MUST be present
+and MUST be marked critical.
+Additionally, the following EKUs MUST NOT be present:
+	• anyExtendedKeyUsage
+	• id-kp-serverAuth
+
+Other values SHOULD NOT be present. If any other value is present, the CA MUST have a
+business agreement with a Platform vendor requiring that EKU in order to issue a
+Platform‐specific code signing certificate with that EKU.
+*/
+
+func init() {
+	lint.RegisterCertificateLint(&lint.CertificateLint{
+		LintMetadata: lint.LintMetadata{
+			Name:          "e_cs_eku_required",
+			Description:   "If the Certificate is a Code Signing Certificate, then id-kp-codeSigning MUST be present. anyExtendedKeyUsage and id-kp-serverAuth MUST NOT be present.",
+			Citation:      "CABF CS BRs 7.1.2.3.f",
+			Source:        lint.CABFCSBaselineRequirements,
+			EffectiveDate: util.CABF_CS_BRs_1_2_Date,
+		},
+		Lint: NewCsEKURequired,
+	})
+}
+
+type csEKURequired struct{}
+
+func NewCsEKURequired() lint.LintInterface {
+	return &csEKURequired{}
+}
+
+func (l *csEKURequired) CheckApplies(c *x509.Certificate) bool {
+	return util.IsCodeSigning(c.PolicyIdentifiers) && util.IsSubscriberCert(c)
```

Comment:
```suggestion
	return util.IsSubscriberCert(c) || util.IsSubCA(c)
```

Reply:
I did add an additional check for if it's a SubCa to also check for emailprotection